util/chunk/chunk.go

```diff
@@ -112,12 +159,68 @@ func (c *Chunk) SetNumVirtualRows(numVirtualRows int) {
 // Reset resets the chunk, so the memory it allocated can be reused.
 // Make sure all the data in the chunk is not used anymore before you reuse this chunk.
 func (c *Chunk) Reset() {
```

Comment:
I think we should keep this function unchanged, for the scenarios that only want to reset content in the `Chunk` without growing.

Reply:
got it~